tests/network/bgp/test_bgp_connectivity.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from libs.net.traffic_generator import is_tcp_connection
+from utilities.virt import migrate_vm_and_verify
+
+pytestmark = [pytest.mark.bgp, pytest.mark.usefixtures("bgp_setup_ready")]
```

Comment:
`pytest.mark.bgp` should be commented out and tests should be quarantined, right?

Reply:
let's leave this comment unresolved then till this change is added